v2/emailpassword/custom-ui/handling-session-tokens.mdx

```diff
@@ -203,6 +203,92 @@ fileprivate class NetworkManager {
 By default our mobile SDKs use a bearer token in the Authorization header to provide credentials.
 :::
 
+### For Flutter
+
+<Question
+    question="Which library are you using for networking?">
+<Answer title="http">
+
+You can make requests as you normally would with `http`, the only difference is that you import the client from the supertokens package instead.
+
+```dart
+// Import http from the SuperTokens package
+import 'package:supertokens_flutter/http.dart' as http;
+
+Future<void> makeRequest() {
+    Uri uri = Uri.parse("http://localhost:3001/api");
+    var response = await http.get(uri);
+    // handle response
+}
+```
+
+<h4>Using a custom http client</h4>
+
+If you use a custom http client and want to use SuperTokens, you can simply provide the SDK with your client. All requests will continue to use your client along with the session logic that SuperTokens provides.
+
+```dart
+// Import http from the SuperTokens package
+import 'package:supertokens_flutter/http.dart' as http;
+
+Future<void> makeRequest() {
+    Uri uri = Uri.parse("http://localhost:3001/api");
+
+    // provide your custom client to SuperTokens
+    var httpClient = http.Client(client: customClient)
+
+    var response = await httpClient.get(uri);
+    // handle response
+}
+```
+
+</Answer>
+<Answer title="Dio">
+
+<h4>Add the SuperTokens interceptor</h4>
+
+```dart
+import 'package:supertokens_flutter/dio.dart';
+
+void setup() {
+    Dio dio = Dio(...)
+    dio.interceptors.add(SuperTokensInterceptorWrapper(client: dio));
+}
+```
+
+Or use the extension method provided by the SuperTokens SDK
+
+```dart
+import 'package:supertokens_flutter/dio.dart';
+
+void setup() {
+  Dio dio = Dio();  // Create a Dio instance.
+  dio.addSupertokensInterceptor();
+}
+```
+
+<h4>Making network requests</h4>
+
+You can make requests as you normally would with `dio`.
+
+```dart
+import 'package:supertokens_flutter/dio.dart';
+
+void setup() {
+    Dio dio = Dio(...)
+    dio.interceptors.add(SuperTokensInterceptorWrapper(client: dio));
```

Comment:
use the new method contributed by the person

Reply:
Ive mentioned both to keep it consistent with the README for supertokens-flutter